qiskit_nature/second_q/algorithms/excited_states_solvers/qeom.py

```diff
@@ -374,8 +425,8 @@ def _build_commutator_routine(
             The indices of the matrix element and the corresponding qubit
             operator for each of the EOM matrices.
         """
-        m_u, n_u, left_op, right_op_1, right_op_2 = params
-        if left_op is None or right_op_1 is None and right_op_2 is None:
+        m_u, n_u, left_op1, right_op_1, right_op_2 = params
```

Comment:
```suggestion
        m_u, n_u, left_op_1, right_op_1, right_op_2 = params
```
Very nitpicky I know.. but you did it so nicely above, too.. :smile: